app/main/views/services.py

```diff
@@ -160,9 +160,12 @@ def update_service(service_id):
         service.copied_to_following_framework = update['copiedToFollowingFramework']
 
     updated_service = update_and_validate_service(service, update)
-    audit_type = (
-        AuditTypes.update_service_admin if request.args.get('user-role') == 'admin' else AuditTypes.update_service
-    )
+    if 'supplierId' in update:
+        audit_type = AuditTypes.update_service_supplier
+    else:
+        audit_type = (
+            AuditTypes.update_service_admin if request.args.get('user-role') == 'admin' else AuditTypes.update_service
+        )
```

Comment:
Is the user allowed to both update the contents of the service and change the supplier at the same time?

We're starting to pack an awful lot of information into this "audit type" field.

Reply:
Hmm good point.

Reply:
I don't have an _answer_ though of course.